apps/chat/src/constants/search.ts

```diff
@@ -13,3 +14,15 @@ export const MODELS_SEARCH_OPTIONS: IFuseOptions<DialAIEntityModel> = {
   isCaseSensitive: false,
   includeScore: false,
 };
+
+export const TOOLSETS_SEARCH_OPTIONS: IFuseOptions<ToolsetModel> = {
+  keys: ['name', 'version'],
```

Comment:
What do you think about making MODELS_SEARCH_OPTIONS more generic instead of creating new one for toolsets (since options are same)?